.github/workflows/ci.yml

```
@@ -15,15 +15,15 @@ concurrency:
 
 jobs:
   macos:
-    name: macOS 13 (Xcode 14.3.1)
-    runs-on: macos-13
+    name: macOS 15 (Xcode 16.4)
+    runs-on: macos-15
     strategy:
       matrix:
         config: ['debug', 'release']
     steps:
       - uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)